.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ common: &common
           - cache-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "tox.ini" }}
     - run:
         name: install dependencies
-        command: pip install --user tox
+        command: ./.circleci/fetch_deps.sh
```

Comment:
For testing speed, maybe a separate dependency install that only triggers on the benchmark job?

```yml
     - run:
         name: install dependencies
         command: pip install --user tox
     - run:
         name: install benchmark dependencies
         command: if [ "$CIRCLE_JOB" == "py36-benchmark" ]; then ./.circleci/fetch_deps.sh; fi
```

Reply:
Yep, that looks good!